### PR TITLE
Set up test suite

### PR DESCRIPTION
**Pull request by LaraSantosMadeTech**
- Installed Jest testing library
- Installed supertest for making an HTTP request 
- Write a passing test
- Revert back to commonjs because Jest only ships with experimental support for ECMAScript Modules (ESM). https://jestjs.io/docs/ecmascript-modules

### CODE REVIEW

File: server.test.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+//const server = require('./server');
```
Comment (Joseaie) [inline server.test.js:1]: Will this be used later on?

**Comment by Joseaie** [comment]
Looks good to me! Just left a comment about a comment 😂 I would also change the title of this PR to something like "Set up testing/ set up test suite" when you go to merge, because it wasn't only Jest that was installed.

File: server.test.js (hunk 1 above)
Comment (kpinakula) [reply server.test.js:1]:
```suggestion
```

Comment (kpinakula) [reply server.test.js:1]: Yes, opinions were divided on leaving it in or not because we'll need it soon.

Comment (Joseaie) [reply server.test.js:1]: In that case, feel free to merge! 😊

**Pull request merged**